api/src/main/java/tennisclub/validators/IsEndTimeLaterThanStartTimeValidator.java

```diff
@@ -0,0 +1,32 @@
+package tennisclub.validators;
+import org.apache.commons.beanutils.PropertyUtils;
+import tennisclub.annotations.IsEndTimeAfterStartTime;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.lang.reflect.InvocationTargetException;
+import java.time.LocalDateTime;
+
+public class IsEndTimeLaterThanStartTimeValidator implements ConstraintValidator<IsEndTimeAfterStartTime, Object> {
```

Comment:
I'd consider a shorter name, maybe something like `EndAfterStartValidator` (or maybe without the `Validator` part) 🤔

Reply:
Oki, will do 😄 I'm just used to more descriptive names 😆